src/style.css

```diff
@@ -1,3 +1,60 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  --active-color: #00acdc;
+  --default-color: #000;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
u can use `*` for `box-sizing: border-box` property to make sure it works for the whole project